README.md

```diff
@@ -57,29 +84,28 @@ command line arguments.
 
 Option | Shortened version | Description | Example Input
 --- | --- | --- | --- 
-`--port` | `-p` | This is a *mandatory* option that defines the port to be used to send API commands to your node | `-p 14800`
+`--port` | `-p` | This is a *mandatory* option that defines the port to be used to send API commands to your node | `-p 14265`
 `--neighbors` | `-n` | Neighbors that you are connected with will be added via this option. | `-n "udp://148.148.148.148:14265 udp://[2001:db8:a0b:12f0::1]:14265"`
 `--config` | `-c` | Config INI file that can be used instead of CLI options. See more below | `-c iri.ini`
-`--udp-receiver-port` | `-u` | UDP receiver port | `-u 14800`
-`--tcp-receiver-port` | `-t` | TCP receiver port | `-t 14800`
+`--udp-receiver-port` | `-u` | UDP receiver port | `-u 14600`
+`--tcp-receiver-port` | `-t` | TCP receiver port | `-t 15600`
 `--testnet` | | Makes it possible to run IRI with the IOTA testnet | `--testnet`
 `--remote` | | Remotely access your node and send API commands | `--remote`
 `--remote-auth` | | Require authentication password for accessing remotely. Requires a correct `username:hashedpassword` combination | `--remote-auth iotatoken:LL9EZFNCHZCMLJLVUBCKJSWKFEXNYRHHMYS9XQLUZRDEKUUDOCMBMRBWJEMEDDXSDPHIGQULENCRVEYMO`
 `--remote-limit-api` | | Exclude certain API calls from being able to be accessed remotely | `--remote-limit-api "attachToTangle, addNeighbors"`
 `--send-limit`| | Limit the outbound bandwidth consumption. Limit is set to mbit/s | `--send-limit 1.0`
 `--max-peers` | | Limit the number of max accepted peers. Default is set to 0 (mutual tethering) | `--max-peers 8`
-`--dns-resolution-false` | | Ignores DNS resolution refreshing  | --dns-resolution-false	
+`--dns-resolution-false` | | Ignores DNS resolution refreshing  | `--dns-resolution-false`	
 ### INI File
 
 You can also provide an ini file to store all of your command line options and easily update (especially neighbors) if needed. You can enable it via the `--config` flag. Here is an example INI file:
 ```
 [IRI]
-PORT = 14700
-UDP_RECEIVER_PORT = 14700
-NEIGHBORS = udp://my.favorite.com:15600
+PORT = 14265
+UDP_RECEIVER_PORT = 14600
+NEIGHBORS = udp://my.favorite.com:14600
 IXI_DIR = ixi
 HEADLESS = true
```

Comment:
I am pretty sure the HEADLESS config is non existent now.
@alon-e, you know what it was for?

anyhows, you can remove it.